src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/BatchMappingUpdateViewModels.cs

```diff
@@ -0,0 +1,80 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Public.Data;
+
+public record BatchLocationMappingUpdatesRequest
+{
+    public List<LocationMappingUpdateRequest> Updates { get; init; } = [];
+
+    public class Validator : AbstractValidator<BatchLocationMappingUpdatesRequest>
+    {
+        public Validator()
+        {
+            RuleForEach(request => request.Updates)
+                .SetValidator(new LocationMappingUpdateRequest.Validator());
+        }
+    }
+}
+
+public record LocationMappingUpdateRequest
+{
+    public GeographicLevel? Level { get; init; }
```

Comment:
should this be non-nullable?

Reply:
I was forced to do this so that it is not auto-allocated the default value for this enum if the user doesn't include it in their request body.  By default it'll choose the first GeographicLevel value in the enum otherwise.

Reply:
ah, I see. That's a good point!